python/tank/platform/qt/tankqdialog.py

```diff
@@ -312,7 +312,12 @@ def _format_context_property(p, show_type=False):
             self.ui.btn_file_system.clicked.connect( self._on_filesystem )
             self.ui.btn_shotgun.clicked.connect( self._on_shotgun )
             self.ui.btn_reload.clicked.connect( self._on_reload )
-            
+
+            # When there is no file system locations, hide "Jump to File System" and "Jump to Shotgun" buttons.
+            if not self._bundle.context.filesystem_locations:
+                self.ui.btn_file_system.setVisible(False)
+                self.ui.btn_shotgun.setVisible(False)
```

Comment:
Why is the jump to shotgun button not visible? in the case no folders exist on disk, can we still keep the shotgun button around so that an artist can jump to the webapp?